packages/components/src/guide/style.scss

```diff
@@ -0,0 +1,119 @@
+.components-guide {
+	.components-modal__header {
+		background: none;
+		border-bottom: none;
+
+		.components-icon-button {
+			align-self: flex-start;
+			margin-top: $grid-size-xlarge;
+			position: static;
+		}
+	}
+
+	&__container {
```

Comment:
While I think this convention certainly makes things more concise, in practice I find it adds difficulty to a very common use-case of searching for a class name in the codebase. To try to search for "components-guide__container" would not yield this as a result. While it requires a couple more characters to type, I personally prefer the fully expanded form in source:

```scss
.components-guide__container {
```